posthog/models/notification_setting.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from typing import Optional
+
+from posthog.models import User
+
+
+class NotificationSetting(Enum):
+    WEEKLY_PROJECT_DIGEST = "weekly_project_digest"
+    PLUGIN_DISABLED = "plugin_disabled"
+    # Add more notification types as needed
+
+
+def should_send_notification(
+    user: User,
+    notification_type: NotificationSetting,
+    team_id: Optional[int] = None,
+) -> bool:
+    """
+    Determines if a notification should be sent to a user based on their notification settings.
```

Comment:
Maybe just added a comment that `get_members_to_notify` also exists. Also the plugin check is a bit more complex with the org role check as well so maybe isn't needed here.

Reply:
I just moved it all to that file directly under that func, it should have been there in the first place